### PR TITLE
fix: Reload loan Table in Salary Slip when change Employee

### PR DESCRIPTION
**Pull request by alaa-alsalehi**
fix: Loans child table in Salary slip

version-13-hotfix

Description: is not reload when the employee changed

Before fix:
Before

After fix:
After
@ruchamahabal @pateljannat

### CODE REVIEW

**Comment by alaa-alsalehi** [comment]
@deepeshgarg007 we rebase the PR can you check please

File: erpnext/payroll/doctype/salary_slip/salary_slip.py [hunk 1]
```diff
@@ -1352,23 +1352,23 @@ def set_loan_repayment(self):
...
-							"loan": loan.name,
-							"total_payment": amounts["interest_amount"] + amounts["payable_principal_amount"],
-							"interest_amount": amounts["interest_amount"],
-							"principal_amount": amounts["payable_principal_amount"],
-							"loan_account": loan.loan_account,
-							"interest_income_account": loan.interest_income_account,
-						},
-					)
+		if self.get("loans"):
+			[self.remove(d) for d in self.get("loans")]
```
Comment (ruchamahabal) [inline erpnext/payroll/doctype/salary_slip/salary_slip.py]:
```suggestion
		self.set("loans", [])
```

**Comment by ruchamahabal** [comment]
@ServeMe can you make the requested change and update the out-of-date branch? I don't have permission to update this branch (since "allow edits for maintainers" seems to be disabled for this PR) so cannot do it myself.

**Comment by alaa-alsalehi** [comment]
@ruchamahabal done

**Review by ruchamahabal — APPROVED** [review]

**Pull request merged**